dockers/docker-ptf/Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN sed --in-place 's/httpredir.debian.org/debian-archive.trafficmanager.net/' /
         python-dev          \
         wget                \
         cmake               \
+	isc-dhcp-server	    \
```

Comment:
DHCP server is used for DHCP testing. One option was to create a seperate Docker container for the server but after discussion with Pavel and Darren it seems ok to just install isc-dhcp-server in the ptf docker. The client (ptf code) and server run on the same docker container.